examples/config.yml

```diff
@@ -3,6 +3,7 @@ defaultActionTimeout: 600
 maximumActionTimeout: 3600
 maxEntrySizeBytes: 2147483648 # 2 * 1024 * 1024 * 1024
 prometheusPort: 9090
+processorsDerive: OSHI
```

Comment:
What if we make this smarter and less complex by doing both and comparing the results? Then picking one we determine to be more accurate (which in this case appears to be OSHI. One thing I'd love to see going forward is to provide for reliable automatic configuration with less manual configs.

Reply:
Yeah, exposing a config is not all that important.  I've removed the config.  Buildfarm chooses the highest value between implementations.  Please re-review!